node/pkg/db/db.go

```diff
@@ -161,7 +170,8 @@ func (d *Database) FindEmitterSequenceGap(prefix VAAID) (resp []uint64, firstSeq
 		}
 
 		// Find min/max (yay lack of Go generics)
-		first := false
+		// TODO: add tests
+		first := true
```

Comment:
Why do we need to change this now?

Reply:
This function is used to get lost messages from other guardians, I misunderstood it at first, I reverted and added some tests in a later commit: https://github.com/alephium/wormhole-fork/pull/10/commits/9853146687a108c023f13c102668f45b237fde70